notifications_utils/logging.py

```diff
@@ -51,24 +51,18 @@ def after_request(response):
         extra_fields = {
             'method': request.method,
             'url': request.url,
-            'status': response.status_code
+            'status': response.status_code,
         }
 
         if 'service_id' in g:
-            extra_fields.update({
-                'service_id': g.service_id
-            })
+            extra_fields['service_id'] = g.service_id
```

Comment:
Are we certain this will not be a list? `.update()` gives us added options that are being removed.

Reply:
Yes.  "extra_fields" is declared as a dictionary right above this block.

Reply:
Not quite what I meant. The [update method](The specified items can be a dictionary, or an iterable object with key value pairs.) gives us the option of passing a dictionary (or other options). Are we certain we are okay losing that functionality?

Reply:
Yes.  "g" is the [Flask global context](https://flask.palletsprojects.com/en/2.3.x/api/#flask.g), and `g.service_id` is a primitive type.
```
...va/notification-api$ grep -rn -e "\bg\.service_id" .
./app/authentication/auth.py:66:        g.service_id = current_app.config.get('ADMIN_CLIENT_USER_NAME')
./app/authentication/auth.py:179:        g.service_id = api_key.service_id
```

Reply:
Does `g.endpoint` behave the same?

Reply:
Yes.  This should just be a URL string.
```
...va/notification-api$ grep -rn -e "\bg\.endpoint" .
./app/__init__.py:406:        g.endpoint = request.endpoint
```

Reply:
Great, thanks!